packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -47,6 +47,16 @@ export class Tabs extends FASTElement {
      */
     @attr
     public activeid: string;
+    public activeidChanged(): void {
+        if (
+            this.$fastController.isConnected &&
+            this.tabs.length <= this.tabpanels.length
+        ) {
+            this.setTabs();
+            this.setTabPanels();
+            this.handleActiveIndicatorPosition();
+        }
```

Comment:
Can you add a couple tests showing that updating the activeId changes it accordingly?

Reply:
I believe there is a test for setting the activeId - we should add one after for updating.

Reply:
Updated